components/TextControls/docs/typical-use.md

```diff
@@ -0,0 +1,3 @@
+### TextFields
+
+Using TextControl textfields is supposed to be as similar to using UITextField as possible. The most crucial difference from a usability standpoint relates to the sizing behavior of these textfields and the implications of that behavior for layout. Where UITextField can be whatever height a user wants it to be, MDCTextControl textfields have heights that they need to be in order to look correct. There is a precise vertical arrangement of the various subviews in MDCTextControl textfields, and this arrangement affects how tall the view ends up being. Other things like font also help determine the height. Ensuring that MDCTextControl textfields have their desired heights is done differently depending on whether one is working in an Auto Layout or a Manual Layout environment. In an Auto Layout environment, the textfield's desired height will be reflected in `intrinsicContentSize`, and the user will not have to do anything to ensure that the correct height is achieved. In a Manual Layout environment, methods like `sizeThatFits:` or `sizeToFit` must be used to inform the frames of the view.
```

Comment:
This section appears to be simply saying to use the typical layout methods for UIKit; is there something new or novel here that warrants being called out? If not, consider deleting this section because we should be able to assume that clients are aware of the need to configure the height of UIKit views using auto layout or `sizeThatFits:`.

Reply:
Unlike with UITextField, one cannot simply set the height to X points and expect it to look good--it's got a particular height that it wants to be. I thought that was worth emphasizing. Other than that, using it is just like using a UITextField.

Reply:
I see; that can likely be emphasized with less text then. Just a sec...

Reply:
> TextControl text fields require a specific minimum size in order to display all content correctly. When using manual layout, please use `sizeThatFits:` or `sizeToFit` to calculate the correct size of the view.

Reply:
> TextControl text fields require a specific minimum size in order to display all content correctly. When using manual layout, please use `sizeThatFits:` or `sizeToFit` to calculate the correct size of the view. AutoLayout will behave as expected.

Called out AutoLayout behavior specifically as well.

Reply:
Do you want me to get rid of what I wrote and use just these two paragraphs you wrote?

Reply:
It's a suggested simplification; if you feel any important information is missing from the suggested edit though please adjust!

Reply:
got it, thanks 👍